app/services/league/utils/scrape.js

```diff
@@ -0,0 +1,25 @@
+const cheerio = require('cheerio');
+const fetch = require('node-fetch');
+const Tooltip = require('../models/tooltipModel');
+
+exports.scrapeWebsite = async () => {
+    const terms = ['previousclose', 'openingprice', 'volume', 'averagedailytradingvolume', 'marketcapitalization', 'beta', 'price-earningsratio', 'eps', 'earnings-announcement', 'costbasis'];
```

Comment:
I think we should just manually update this and run the script if we need new tooltips. Especially since I don't see how the frontend will store a tooltip that will later be visible without us manually placing